website/project/views/node.py

```diff
@@ -969,7 +969,7 @@ def get_summary(**kwargs):
 
     auth = kwargs['auth']
     node = kwargs['node'] or kwargs['project']
-    rescale_ratio = kwargs.get('rescale_ratio')
+    rescale_ratio = kwargs.get('rescale_ratio') or float(request.args.get('rescale_ratio'))
```

Comment:
If `kwarsg.get('rescale_ratio')` and `request.args.get('rescale_ratio')` are both `None`, `float()` will raise a `TypeError` if the value of `request.args['rescale_ratio']` cannot be converted to a `float`, a `ValueError` is raised. 

These cases must be handled.